src/planscape/users/views.py

```diff
@@ -59,3 +62,29 @@ def is_verified_user(request: HttpRequest) -> HttpResponse:
         return JsonResponse({"verified": True})
     except Exception as e:
         return HttpResponseBadRequest("Ill-formed request: " + str(e))
+
+
+def verify_password_reset_token(
+    request: HttpRequest, user_id: str, token: str
+) -> HttpResponse:
+    # Get the UserModel
+    UserModel = get_user_model()
+
+    if "allauth" in settings.INSTALLED_APPS:
```

Comment:
do we anticipate not having allauth?

Reply:
No, this is mirroring what is in the dj-rest-auth code for validating tokens though:
https://github.com/iMerica/dj-rest-auth/blob/master/dj_rest_auth/serializers.py#L276-L291

I thought it may be best to follow their implementation. 

Let me know if you would prefer I cut it down to just the allauth path.

Reply:
Let's keep both; I see in settings.py what appears to be two auth mechanisms.  However, put a comment here why we have the non-allauth code here as well.

Reply:
Added a comment.